app/assets/src/components/views/amr_heatmap/AMRHeatmapView.jsx

```diff
@@ -79,8 +135,9 @@ export default class AMRHeatmapView extends React.Component {
       <div className="row visualization-content">
         <ErrorBoundary>
           <AMRHeatmapVis
-            samplesWithAMRCounts={this.state.samplesWithAMRCounts}
-            selectedOptions={this.state.selectedOptions}
+            // maxValueUpdater={this.updateMaxValue}
```

Comment:
Nit: is this comment supposed to be here?

Reply:
removed!